frontend/src/utilities/js-message-dispatcher.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { reactive } from "vue";
+import { plainToInstance } from "class-transformer";
```

Comment:
Nitpick: please put a separator newline between the first two imports (of libraries) and the other imports (of source code files).